hw/ip/usbdev/dv/cov/usbdev_unr_excl.el

```diff
@@ -2,6 +2,37 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
+// The following exclusions have been added manually on account of the dynamic behavior
+// and understanding of the USB and USBDEV being unavailable to UNR tooling.
```

Comment:
I think we place manual exclusions usually in a different file than UNR-generated ones -- so that when re-running UNR one can simply override the contents of one file (except for the license header). @rswarbrick WDYT?

Reply:
I would have done that, but was mimicking what's been done in rv_dm

Reply:
Separated out, since that's my preference too, and rechecked.